package.json

```diff
@@ -25,35 +24,28 @@
   },
   "dependencies": {
     "@material-ui/core": "^3.9.1",
-    "@material-ui/icons": "^3.0.2",
     "lodash-es": "^4.17.11",
-    "material-ui-pickers": "^2.1.2"
+    "material-ui-pickers": "^2.1.2",
+    "react-jss": "^8.6.1"
```

Comment:
`react-jss` was being bundled together when using `rollup`. With this approach, we make it a requirement, but other bundlers will be able to tree-shake it away.

Reply:
Fair enough, we have a hard dependency on it in `ThemeProvider.jsx`.